packages/amplify-cli-core/src/category-interfaces/category-override-base.ts

```diff
@@ -0,0 +1,12 @@
+import { $TSContext } from '..';
+import { Template } from './amplifyBaseCDKTypes';
+
+export abstract class AmplifyCategoryTransform {
+  _resourceName: string;
+  constructor(resourceName: string) {
+    this._resourceName = resourceName;
+  }
+  abstract transform(context: $TSContext): Promise<Template>;
+  abstract applyOverride(): Promise<void>;
+  abstract saveBuildFiles(context: $TSContext, template: Template): Promise<void>;
```

Comment:
Could you please add a comment on which files will be generated from saveBuildFiles.

Reply:
done